.github/workflows/autoupdate.md

```diff
@@ -0,0 +1,7 @@
+# autorebase.yml
+
+Operates on open PRs and automatically merges updates on target branches into the PR branch
+
+# TODO
+
+We already have `autorebase.yml`. Why is this still needed?
```

Comment:
autoupdate creates merge commits while autorebase does a rebase

Reply:
Any reason why we need both approaches? They seem to solve the same problem.

Reply:
functionally they're different and then it comes down to preference

Reply:
?? Aren't both run on every push to develop?
And whose preference, I mean how does one get to choose?

Reply:
Autoupdate specifies to only act if the `autoupdate:opt-in` label is set. Potentially there is another label to activate the autorebase. This should be documented somewhere and potentially be added to the PR template if we want contributors to make use of these.

Reply:
they always run but they only do something if the PR is labelled accordingly. the label names are `auto{rebase,update}:opt-in` respectively, and can be applied to a PR by the author or maintainers. one will rebase source onto target and the other will merge target into source.